schemas/IDLs/org.gel.models.report.avro/5.0.0-SNAPSHOT/InterpretationRequestRD.avdl

```diff
@@ -31,34 +31,24 @@ protocol InterpretationRequestsRD {
         ReportVersionControl versionControl;
 
         /**
-        Unique id for this report
+        id for this request
         */
         string interpretationRequestId;
```

Comment:
I may be being picky, but could we think of having generic names (ie: `id` instead of `interpretationRequestId`) in fields like this?